database/seeders/UserSeeder.php

```diff
@@ -20,7 +20,7 @@ public function run()
             'firstname' => 'Admin',
             'lastname' => 'Admin',
             'email' => 'admin@orfed.com.br',
-            'password' => '$2y$10$RiUxrJkNYS9x0WZ61FQYCOw.3qo.4TWCSSIsZqyfz4XuMuIB14Bta',
+            'password' => bcrypt('admin'),
```

Comment:
isso aqui nao da certo porque na model tem o setPasswordAttribute que já faz o bcript
```suggestion
            'password' => 'admin',
```

Reply:
Aproveitei pra adicionar uma mutator para o password, assim quando se usar a model de user para criar um usuario, não precisa hashear o password, basta apenas passalo